### ISSUE TITLE
Percona GPG key has changed

### ISSUE THREAD

**Issue by mamedin** [artefactual-labs/ansible-percona#30]
Running the role on Bionic the GPG fails:

```
TASK [artefactual.percona : Adding percona repository] ***************************************************************************************************************************************
fatal: [001-atom]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "module_stderr": "Shared connection to 35.177.97.128 closed.\r\n", 
    "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_6D0ouN/ansible_module_apt_repository.py\", line 556, in <module>\r\n    main()\r\n  File \"/tmp/ansible_6D0ouN/ansible_module_apt_repository.py\", line 544, in main\r\n    cache.update()\r\n  File \"/usr/lib/python2.7/dist-packages/apt/cache.py\", line 559, in update\r\n    raise FetchFailedException(e)\r\napt.cache.FetchFailedException: W:GPG error: http://repo.percona.com/apt bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 9334A25F8507EFA5, E:The repository 'http://repo.percona.com/apt bionic InRelease' is not signed.\r\n", 
    "rc": 0
}
```

**Comment by mamedin** [comment]
More info: TASK [artefactual.percona : Adding percona repository] ***************************************************************************************************************************************
fatal: [001-atom]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "module_stderr": "Shared connection to 35.177.97.128 closed.\r\n", 
    "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_6D0ouN/ansible_module_apt_repository.py\", line 556, in <module>\r\n    main()\r\n  File \"/tmp/ansible_6D0ouN/ansible_module_apt_repository.py\", line 544, in main\r\n    cache.update()\r\n  File \"/usr/lib/python2.7/dist-packages/apt/cache.py\", line 559, in update\r\n    raise FetchFailedException(e)\r\napt.cache.FetchFailedException: W:GPG error: http://repo.percona.com/apt bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 9334A25F8507EFA5, E:The repository 'http://repo.percona.com/apt bionic InRelease' is not signed.\r\n", 
    "rc": 0
}

**Comment by sevein** [comment]
Fixed by #31.
